autofs-5.0.5 - add locality as valid ldap master map attribute

From: Ian Kent <raven@themaw.net>

The master map dn string parsing is quite strict so we will need to add
allowable attributes as required. One such attribute is "l", the locality.
---

 CHANGELOG        |    1 +
 lib/master_tok.l |    3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)


--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -25,6 +25,7 @@
 - add missing sasl mutex callbacks.
 - fix get query dn failure.
 - fix ampersand escape in auto.smb.
+- add locality as valid ldap master map attribute.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/lib/master_tok.l
+++ autofs-5.0.5/lib/master_tok.l
@@ -110,7 +110,8 @@ AT_OU		([oO][[uU])
 AT_DC		([dD][[cC])
 AT_O		([oO])
 AT_C		([cC])
-DNATTRSTR	({AT_CN}|{AT_NMN}|{AT_AMN}|{AT_OU}|{AT_DC}|{AT_O}|{AT_C})
+AT_L		([lL])
+DNATTRSTR	({AT_CN}|{AT_NMN}|{AT_AMN}|{AT_OU}|{AT_DC}|{AT_O}|{AT_C}|{AT_L})
 DNNAMESTR1	([[:alnum:]_.\- ]+)
 DNNAMESTR2	([[:alnum:]_.\-]+)
 
